### PR TITLE
Add an example for using input arguments with cypher

### LINKED ISSUE #110: Cypher Directive documentation improvements
Some feedback from some lacking documentation regarding `@cypher` directive:

It is not clear in the doc how to add a param in a GraphQL query directly which then binds to a variable in a cypher query through the `@cypher` directive.

Add example and information of a cypher query with the `@cyper` directive doing a lot of matches with several properties from different nodes and relationships, then in order for GraphQL to parse it properly you need to create a typeDef which holds all that information of all the properties you want to return, and then return them as a map.

### PR DESCRIPTION
**Pull request by angrykoala**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-132.surge.sh

Updated pages:
- [<code>@cypher</code>](https://neo4j-docs-graphql-132.surge.sh/graphql/5/type-definitions/directives/cypher)

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc [hunk 1]
```diff
@@ -154,6 +154,26 @@ type Query {
...
+The `@cypher` statement can access the query parameters by prepending `$` to the parameter name. For example:
+
+[source, graphql, indent=0]
+----
+type Query {
+    name(value: String): String @cypher(statement: "RETURN $value AS res", columnName: "res")
+}
+----
+
+The following GraphQL query will return the parameter  `value`:
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/cypher.adoc]:
```suggestion
The following GraphQL query returns the parameter `value`:
```

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc [hunk 2]
```diff
@@ -154,6 +154,26 @@ type Query {
...
+
+[source, graphql, indent=0]
+----
+query {
+  name(value: "aca")
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/cypher.adoc]: what's "aca"?

Comment (angrykoala) [reply modules/ROOT/pages/type-definitions/directives/cypher.adoc]: It's a random text. I will change for something better

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-132.surge.sh

Updated pages:
- [<code>@cypher</code>](https://neo4j-docs-graphql-132.surge.sh/graphql/5/type-definitions/directives/cypher)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-132.surge.sh

Updated pages:
- [<code>@cypher</code>](https://neo4j-docs-graphql-132.surge.sh/graphql/5/type-definitions/directives/cypher)

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc [hunk 3]
```diff
@@ -154,6 +154,26 @@ type Query {
...
+    name(value: String): String @cypher(statement: "RETURN $value AS res", columnName: "res")
+}
+----
+
+The following GraphQL query returns the parameter `value`:
+
+[source, graphql, indent=0]
+----
+query {
+  name(value: "Arthur")
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/cypher.adoc]: I found this name in the Queries page:

```suggestion
  name(value: "Jane Smith")
```

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc [hunk 4]
```diff
@@ -154,6 +154,26 @@ type Query {
...
+The `@cypher` statement can access the query parameters by prepending `$` to the parameter name. For example:
+
+[source, graphql, indent=0]
+----
+type Query {
+    name(value: String): String @cypher(statement: "RETURN $value AS res", columnName: "res")
+}
+----
+
+The following GraphQL query returns the parameter `value`:
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/cypher.adoc:167]: is the parameter `name` or `value`? i think i'm confused

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc [hunk 5]
```diff
@@ -154,6 +154,26 @@ type Query {
...
+
+[source, graphql, indent=0]
+----
+query {
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/cypher.adoc:171]: another point of confusion on my part: how does this query relate to the query above with the cypher directive in it?

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc (hunk 4 above)
Comment (angrykoala) [reply modules/ROOT/pages/type-definitions/directives/cypher.adoc:167]: Here I'm trying to reference the parameter (`value`) passed to the query (`name`).

File: modules/ROOT/pages/type-definitions/directives/cypher.adoc (hunk 5 above)
Comment (angrykoala) [reply modules/ROOT/pages/type-definitions/directives/cypher.adoc:171]: In this query we are passing a parameter called `value` that is caught by the `@cypher` directive (see the `$value`)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by rsill-neo4j — APPROVED** [review]

**Pull request merged**